### PR TITLE
feat: access section to control captive-portal

### PR DESCRIPTION
**Pull request by luandro**
Adds Pirania administrator section to Lime-App.

@nicopace @gmarcos87 squashed all commits for non-polution of git history.

(I accidently did a push `git push --force` to `pirania` branch, so lost all my history :cry: )

### CODE REVIEW

File: i18n/translations/pt.json [hunk 1]
```diff
@@ -4,6 +4,76 @@
 			"Athena in Wonderland"
 		]
 	},
+	"access_9a94f04e": "Accesso",
+  "active_member_vouchers_1d7e4f0d": "Códigos de membros ativos",
```
Comment (nicopace) [inline i18n/translations/pt.json:8]: Looks like from now on the spacing is wrong. Please correct

File: plugins/lime-plugin-admin/src/adminEpics.js [hunk 2]
```diff
@@ -18,10 +18,10 @@ import { from } from 'rxjs';
 import { ofType } from 'redux-observable';
 import { mergeMap, map, catchError, delay } from 'rxjs/operators';
 
-const loginAction = ( action$, store, { wsAPI } ) =>
```
Comment (nicopace) [inline plugins/lime-plugin-admin/src/adminEpics.js:21]: no real change in this file, just style change. please remove or put aside.

File: plugins/lime-plugin-align/src/alignEpics.js [hunk 3]
```diff
@@ -85,4 +85,4 @@ const getSignal = ( action$, state$, { wsAPI } ) =>
 		catchError(e => [{ type: SIGNAL_GET_FAILD }])
 	);
 
-export default { ifaceLoad, allStationsLoad, ifaceChange, initAlign, getSignal };
```
Comment (nicopace) [inline plugins/lime-plugin-align/src/alignEpics.js:88]: no real change in this file, please remove the change

File: plugins/lime-plugin-core/src/metaEpics.js [hunk 4]
```diff
@@ -82,7 +82,7 @@ const loadNetwork = ( action$, store, { wsAPI }) =>
 		mergeMap(() => getCloudNodes(wsAPI,store.value.meta.sid)),
 		map(payload => ({ type: CONECTION_LOAD_NEIGHBORS_SUCCESS, payload }))
 	);
-
```
Comment (nicopace) [inline plugins/lime-plugin-core/src/metaEpics.js:85]: no real change in this file, please remove the change

File: plugins/lime-plugin-metrics/src/metricsEpics.js [hunk 5]
```diff
@@ -74,12 +74,12 @@ const loadGatewayMetrics = ( action$, store, { wsAPI }) =>
 			return [{ type: LOAD_METRICS_GATEWAY_SUCCESS, payload: {} }];
 		})
 	);
-
```
Comment (nicopace) [inline plugins/lime-plugin-metrics/src/metricsEpics.js:77]: no real change in this file, please remove the change

File: plugins/lime-plugin-rx/src/rxEpics.js [hunk 6]
```diff
@@ -23,17 +23,17 @@ import {
 
 import { interval, from } from 'rxjs';
 import { ofType } from 'redux-observable';
-import { map, takeUntil, mergeMap, catchError,switchMap } from 'rxjs/operators';
```
Comment (nicopace) [inline plugins/lime-plugin-rx/src/rxEpics.js:26]: no real change in this file, please remove the change

File: src/config.js [hunk 7]
```diff
@@ -16,7 +18,8 @@ export const plugins = [
 	Metrics,
 	Notes,
 	Meta,
-	//GroundRouting,
```
Comment (nicopace) [inline src/config.js:19]: this is not a real change, please remove

File: src/config.js [hunk 8]
```diff
@@ -6,6 +6,8 @@ import Notes from '../plugins/lime-plugin-notes';
 import Rx from '../plugins/lime-plugin-rx';
 import Admin from '../plugins/lime-plugin-admin';
 import Fbw from '../plugins/lime-plugin-fbw';
+import Pirania from '../plugins/lime-plugin-pirania';
+
```
Comment (nicopace) [inline src/config.js:10]: this is not a real change, please remove

File: src/store.js [hunk 9]
```diff
@@ -24,8 +24,8 @@ export const store = createStore({},rootEpics,rootReducers, new UhttpdService())
 
 // Init websocket
 export const initStore = () => store.dispatch({
-    type: 'meta/CONECTION_START',
```
Comment (nicopace) [inline src/store.js:27]: this is not a real change, please remove

File: src/utils/uhttpd.service.js [hunk 10]
```diff
@@ -1,44 +1,44 @@
 import axios from 'axios';
 import { from } from 'rxjs';
 
-String.prototype.hashCode = function() {
```
Comment (nicopace) [inline src/utils/uhttpd.service.js:4]: all changes in this file are just formatting, please remove this change.

File: src/utils/webSockets.service.js [hunk 11]
```diff
@@ -4,78 +4,80 @@ import { Observable } from 'rxjs';
 import { map, filter } from 'rxjs/operators';
```
Comment (nicopace) [inline src/utils/webSockets.service.js:4]: there doesn't seem to be any relevant change on this file, please remove this change.

File: src/utils/webSockets.observable.js [hunk 12]
```diff
@@ -1,4 +1,4 @@
-import { Observable } from 'rxjs/Observable';
```
Comment (nicopace) [inline src/utils/webSockets.observable.js:1]: You have been changing this import all around the file, but there doesn't seem to exist any change on the rxjs version... was it wrong all the time?

**Review by nicopace — CHANGES_REQUESTED** [review]
Couldn't review functionality, but did found some things that need to be changed in order for the PR to be accepted. Please let me know if you have any questions.

**Comment by luandro** [comment]
@nicopace all those changes are related to linting. I can't unlint.

**Comment by nicopace** [comment]
> @nicopace all those changes are related to linting. I can't unlint.

There are two ways it could be fixed IMHO:
* You can remove them from the PR, and put them aside.
* Or you can do one commit with all the linting alltogether.

**Pull request closed without merging**